### PR TITLE
Remove worker service, and use pod ip instead.

### PR DESCRIPTION
**Pull request by skydoorkai**

### CODE REVIEW

File: elasticdl/python/master/k8s_instance_manager.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ def __init__(
 
         # Protects followed variables, which are accessed from event_cb.
         self._lock = threading.Lock()
-        # worker id to (pod name, phase) mapping
+        # worker id to (pod name, ip, phase) mapping
```
Comment (workingloong) [inline elasticdl/python/master/k8s_instance_manager.py:104]: We can get pod by pod name and get IP by `pod.status.IP`. Should we need to set IP here?

File: elasticdl/python/master/k8s_instance_manager.py [hunk 2]
```diff
@@ -359,30 +369,34 @@ def _event_cb(self, event):
 
     def get_alive_workers(self):
         alive_workers = []
-        for pod_name, phase in self._worker_pods_phase.values():
+        for pod_name, _, phase in self._worker_pods_ip_phase.values():
```
Comment (workingloong) [inline elasticdl/python/master/k8s_instance_manager.py:372]: We can get pod IP by 
```python
pod = k8s_client.get_pod(pod_name)
ip = pod.status.pod_ip
```
https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/V1PodStatus.md

Comment (skydoorkai) [reply elasticdl/python/master/k8s_instance_manager.py:372]: After we record pod_ip, we don't need to call k8s API again.

**Review by workingloong — APPROVED** [review]
LGTM

**Pull request merged**